docs/how-to/external-executor.md

```diff
@@ -10,7 +10,9 @@ Executor be used by different Flows.
 ````{admonition} Where can external Executors run?
 :class: hint
 External Executors can run anywhere from the same environment as the Flow, to a Docker container, or even a remote
-machine.
+environment, such as [JCloud](https://docs.jina.ai/fundamentals/jcloud/).
+
+To deploy external Exectuors on JCloud, please follow [this documentation](https://docs.jina.ai/fundamentals/jcloud/advanced/#external-executors).
```

Comment:
Instead of passing the absolute uri, please pass this using `{ref}` syntax. [Example](https://github.com/jina-ai/jina/blob/master/docs/fundamentals/jcloud/faq.md#:~:text=Flows%20are%20terminated%20if%20they%20are%20not%20serving%20requests%20for%20the%20last%2024hrs.%20But%20this%20is%20configurable%20by%20passing%20%7Bref%7Dretention%2Ddays%20%3Cretention%2Ddays%3E%20argument.).